tests/plugins/colordialog/manual/addhash.md

```diff
@@ -0,0 +1,13 @@
+@bender-tags: tc, feature, colordialog, 565, 4.8.0
+@bender-ui: collapsed
+@bender-ckeditor-plugins: table,tabletools,colordialog,toolbar,wysiwygarea,sourcearea,contextmenu
+
+1. Right click into table cell.
+1. Select `Cell -> Cell Properties`.
+1. Choose background color.
```

Comment:
Does it mean to click `Choose` button and then type color in a `colordialog`? It might be a little unclear, and typing color without hash in `Cell Properties` dialog does not add hash (and color when closing dialog) so it may seem it is broken in this case.
To make it more clear you may use: `Click Choose next to Background Color` or something similar.